src/lib/klayr/klayr-utils.ts

```diff
@@ -232,32 +232,37 @@ type EstimateFeeParams = {
  * @param params Transaction params
  */
 export function estimateFee(params?: EstimateFeeParams) {
-  const {
-    amount = '1',
-    keyPair = KLY_DEMO_ACCOUNT.keyPair,
-    recipientAddress = KLY_DEMO_ACCOUNT.address,
-    data = '',
-    nonce = 0,
-    isNewAccount
-  } = params || {}
+  try {
```

Comment:
1. Use try/catch inside the `kly/fee` getter:

https://github.com/Adamant-im/adamant-im/blob/fe08d661dcb21eb6a253791f0246fafb72adf77b/src/store/modules/kly/kly-getters.js#L8

2. Rollback changes in the current file